resources/views/auth/register.html.twig

```diff
@@ -106,9 +106,9 @@
 										<span>{{ trans('forms.required') }}</span>
 										<div>
 											<input id="password" required type="password" name="password" autocomplete="off"/>
-											<button type="button">
-												<i class="fas fa-eye"></i>
-												<i class="fas fa-eye-slash"></i>
+											<button class="showPassword" aria-label={{register.show_password}} aria-pressed="false" type="button">
+												<i class="fas fa-eye" aria-hidden="true"></i>
+												<i class="fas fa-eye-slash" aria-hidden="true"></i>
```

Comment:
You've set aria-hidden only on these icons. Should we set aria-hidden on the other show password buttons as well?

Reply:
Yes, missed that thanks